backend/fregepoc/analyzers/core/__init__.py

```diff
@@ -1,8 +1,9 @@
 from .cpp import *  # noqa
+from .golang import *  # noqa
```

Comment:
Actually, one more minor thing - please, make those imports absolute and add a specific error code to ignore in `# noqa` for all the imports.

Reply:
I made imports absolute, and removed `#noqa` altogether.

Reply:
@vladfedoriuk if you think that no additional changes are needed, please merge these changes :)